packages/package_bumbershoot_3_0_8738/.shed.yml

```diff
@@ -1,11 +0,0 @@
-categories:
-- Tool Dependency Packages
-description: Contains a tool dependency definition that downloads the Bumbershoot tools.
-long_description: |
```

Comment:
:+1: So OK to delete the packages? I think that actually applies for package_proteowizard too which will be a separate PR.